content/docs/contributor/anteaterapi/setup.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Setting up to contribute to Anteater API
```

Comment:
Consider shortening title? Something along the lines of "Getting Started".

Additional details should go in the description field.